src/components/plots/Header.jsx

```diff
@@ -90,8 +90,8 @@ const Header = (props) => {
     getFromApiExpectOK,
   );
 
-  if (!data) {
-    return <Skeleton.Input style={{ width: 200 }} active />;
+  if (!data || !config) {
+    return <Skeleton active paragraph={{ rows: 1 }} title={{ width: 500 }} />;
```

Comment:
Similarly here. While `config` is loading we display a skeleton so we can render the page earlier.